src/main/java/core/basesyntax/service/report/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.report;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition.